tests/utilities/test_fetching.py

```diff
@@ -189,19 +192,19 @@ def test_trainer_num_prefetch_batches(tmpdir):
         t0 = time()
         trainer = Trainer(**trainer_kwargs)
         trainer.fit(model)
+        assert isinstance(trainer.data_connector.train_data_fetcher, InterBatchParallelDataFetcher)
```

Comment:
Why move the assertions into the timings?

Reply:
Moved out !